### DISCUSSION TITLE
What's the difference of gk and k?

### DISCUSSION THREAD

**Discussion by dlintw** [helix-editor/helix#9031] [Q&A] [answered]
I don't know what's the "Move up" `gk` meaning.  Seems can not work.

For example, I use `vxxx` marked 3 lines, I wish to move those up. it can not work.

**Comment by gabydd** [comment, marked as answer]
The difference of ``gk`` and ``k`` is how they behave with soft wrapped lines ``k`` will go up one visual line whereas ``gk`` will always go to the actual documents line above

**Reply by dlintw** [reply]
It is better say the difference in the tutor.  And vim migration page.
